model/src/main/java/org/cbioportal/model/AlterationCountByGene.java

```diff
@@ -28,12 +28,12 @@ public void setHugoGeneSymbol(String hugoGeneSymbol) {
 		this.hugoGeneSymbol = hugoGeneSymbol;
 	}
 
-	public Integer getCountByEntity() {
-		return countByEntity;
+	public Integer getCountByCase() {
```

Comment:
sorry I missed this on first pass, but I think `countByCase` is a confusing name. is it different from `alteredCount`?

Reply:
what is `countByCase`? is that the number of altered cases? i think its a confusing name

Reply:
@adamabeshouse we're thinking of renaming this to `numberOfAlteredCases` or `alteredCaseCount` or `alteredCount`? We're thinking this variable is storing the number of "cases/entities" which have one or more alterations in a gene. 

Do you have a preference for any of the above?

Reply:
`numberOfAlteredCases` sounds perfect. imo its not too long, and it describes it completely